app/models/submission.rb

```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 
 class Submission < ApplicationRecord
+  enum :status, { draft: 0, submitted: 1 }
+  enum :judging_status, { not_selected: 0, selected: 1, qualified: 2, winner: 3 }
```

Comment:
for backwards compatibility with the Phoenix app, these need to remain as strings that have the same values, which is possible like this...
```suggestion
  enum :status, { draft: "draft", submitted: "submitted" }
  enum :judging_status, { not_selected: "not_selected", selected: "selected", qualified: "qualified", winner: "winner" }
```